lively.ide/js/inspector/index.js

```diff
@@ -492,25 +500,27 @@ export class PropertyControl extends DraggableTreeLabel {
   }
 
   static renderLayoutControl (args) {
-    let propertyControl; const { target, fastRender, valueString, keyString, value } = args;
+    const { valueString, value } = args;
     return [
       ...this.renderGrabbableKey(args),
       ` ${value ? valueString : 'No Layout'}`, {
         onMouseDown: (evt) => {
-
+          // TODO: add layout popup?!
         }
       }];
   }
 
   static renderColorControl (args) {
-    let propertyControl; const {
-      node, gradientEnabled, fastRender,
-      valueString, keyString, value, target
+    const {
+      node, gradientEnabled,
+      valueString, keyString, value, target, tree
     } = args;
+    const inspector = tree.owner;
```

Comment:
I think we want the view model here too.

Reply:
I see that this works by exposing the viewModel property, but still. I would want to have a consistent access.

Reply:
Good catch. Thanks!